fitbenchmarking/parsing/tests/test_parse_sasview_data.py

```diff
@@ -7,12 +7,6 @@
 
 # Delete four lines below when automated tests are enabled
```

Comment:
Does this comment relate to the lines removed here?

Reply:
Yup! It has been removed in 27aa4c2